### ISSUE TITLE
Release Human-AGVQA dataset and GHVQ metric on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [zczhang-sjtu/GHVQ#4]
Hi @zczhang-sjtu 🤗

I'm Niels and work as part of the open-source team at Hugging Face. I discovered your work through Hugging Face's daily papers as yours got featured: https://huggingface.co/papers/2411.16619.
The paper page lets people discuss about your paper and lets them find artifacts about it (your dataset or metric for instance),
you can also claim the paper as yours which will show up on your public profile at HF, add Github and project page URLs.

I saw in your abstract and GitHub README that the Human-AGVQA dataset and GHVQ metric *will be released* in public.
It'd be great to make these artifacts available on the 🤗 hub, to improve their discoverability/visibility.
We can add tags so that people find them when filtering https://huggingface.co/datasets (for the dataset) and potentially hosting the GHVQ metric if it's a runnable model.

## Uploading dataset (Human-AGVQA)

Would be awesome to make the Human-AGVQA dataset available on 🤗 , so that people can do:

```python
from datasets import load_dataset

dataset = load_dataset("your-hf-org-or-username/Human-AGVQA")
```
See here for a guide: https://huggingface.co/docs/datasets/loading.
Besides that, there's the [dataset viewer](https://huggingface.co/docs/hub/en/datasets-viewer) which allows people to quickly explore the first few rows of the data in the browser.

## Uploading metric (GHVQ)

If the GHVQ metric can be released as a callable model or a self-contained evaluation script, we could also host it on the Hub. See here for a guide: https://huggingface.co/docs/hub/models-uploading.

In this case, we could leverage the [PyTorchModelHubMixin](https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin) class which adds `from_pretrained` and `push_to_hub` to any custom `nn.Module`. Alternatively, one can leverages the [hf_hub_download](https://huggingface.co/docs/huggingface_hub/en/guides/download#download-a-single-file) one-liner to download a checkpoint from the hub.

We encourage researchers to push each model checkpoint (or metric implementation) to a separate model repository, so that things like download stats also work. We can then also link the checkpoints to the paper page.

Let me know if you're interested/need any guidance regarding this!

Kind regards,

Niels
ML Engineer @ HF 🤗
